lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -168,15 +168,15 @@ Function Join-Workgroup {
 
         # 2012+ call the Workgroup arg WorkgroupName, but seem to accept
         try {
-            $rc_result = Remove-Computer -Workgroup $workgroup_name -Credential $domain_cred -Force
+            Remove-Computer -Workgroup $workgroup_name -Credential $domain_cred -Force
         } catch {
             Fail-Json -obj $result -message "failed to remove computer from domain: $($_.Exception.Message)"
         }
     }
 
     # we're already on a workgroup- change it.
     Else {
-        $swg_result = Set-Workgroup $workgroup_name
+        Set-Workgroup $workgroup_name > $null
```

Comment:
So now that we've cleared up the confusion and found that the return statement in `Set-Workgroup` isn't needed you can remove the `> $null` part from here.

Reply:
done